CHANGELOG.md

```diff
@@ -7,6 +7,55 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 <!-- towncrier release notes start -->
 
+## [v4.0.0-rc.0]
+
+### Released 2023-09-26
+
+### Migration from v3
+
+See the [migration guide][v4_migration_guide] for details.
+
+### Breaking Changes
+
+- feat!: remove support for fluent-bit and fluentd [#3244]
+- feat!: truncate fullname after 22 characters [#3248]
+- feat(metrics)!: use otel by default [#3284]
+
+### Added
+
+- feat(logs): add `sumologic.logs.additionalFields` property [#3286]
+- feat(metrics): add additionalServiceMonitors setting [#3292]
+
+### Changed
+
+- feat(prometheus): Removing prometheus recording rules [#3211]
+- feat(metrics): move app metrics filtering to metadata layer [#3232]
+- feat!: use OTLP sources by default [#3297]
+- feat!(metrics): move extra processors after sumologic_schema [#3306]
```

Comment:
These should be under breaking changes. I think we mislabelled the changelog files.

Reply:
There were also some breaking changes under `Fixed` section, I moved them as well.